### PR TITLE
consumer control code: update link to HID usage tables

### PR DESCRIPTION
**Pull request by schneider42**
The old file was not available anymore. I believe this is where it was pointing at in the document.

### CODE REVIEW

File: adafruit_hid/keycode.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Keycode:
     """USB HID Keycode constants.
 
     This list is modeled after the names for USB keycodes defined in
-    https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=53.
```
Comment (dhalbert) [inline adafruit_hid/keycode.py:36]: I pasted https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=53 and it went to the document and page I expected (screenshot below). Could you check this again?

image

File: adafruit_hid/keycode.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Keycode:
...
+    https://www.usb.org/sites/default/files/hut1_21_0.pdf#page=118.
```
Comment (dhalbert) [inline adafruit_hid/keycode.py:36]: This is the newer version of the other document, and the page. It's the Keyboard/Keypad Page (0x07). 118 is the Consumer Control codes page.

**Comment by schneider42** [comment]
Oh, I'm sorry I modified the wrong file. The consumer control codes point to http://www.usb.org/developers/hidpage/Hut1_12v2.pdf#page=75 which is not available (anymore?). I'll update the PR.

**Comment by schneider42** [comment]
Ah looks like they changed name and location of that PDF.

The version referenced in the file now lives under https://www.usb.org/sites/default/files/documents/hut1_12v2.pdf#page=75

The latest version seems to be https://www.usb.org/sites/default/files/hut1_21_0.pdf#page=118

Let me know which one you prefer.

**Comment by dhalbert** [comment]
Certainly let's use the latest version of the document. But could you fix both links, both the keyboard codes and the consumer control codes? The page in the new document for keyboard codes is https://www.usb.org/sites/default/files/hut1_21_0.pdf#page=82.

> Oh, I'm sorry I modified the wrong file. The consumer control codes point to http://www.usb.org/developers/hidpage/Hut1_12v2.pdf#page=75 which is not available (anymore?). I'll update the PR.

**Comment by schneider42** [comment]
Will do

**Comment by steamfoundry** [comment]
Darn.  I found this in the middle of submitting a pull request for the same issue.  

The Keyboard codes are also on PDF page 83, print page 82.   https://www.usb.org/sites/default/files/hut1_21_0.pdf#page=83

**Comment by silverman** [comment]
This seems to have lost steam. I suggest that this is what needs to be done:

1. The URL in the `ConsumerControlCode` docstring should be changed to `https://usb.org/sites/default/files/hut1_21_0.pdf#page=118`, which is where section 15 'Consumer Page' begins.
1. The URL in the `Keycode` docstring should be changed to `https://usb.org/sites/default/files/hut1_21_0.pdf#page=83`, which is where section 10 'Keyboard/Keypad Page' begins.

The commit of @schneider42 already addresses the first one.

**Review by dhalbert — APPROVED** [review]
Thanks @schneider42, @steamfoundry, and @silverman for your work and comments. I consolidated everything, fixed both links, and merged from upstream.

**Pull request merged**